assets/js/ModuleCategory.test.js

```diff
@@ -134,48 +201,74 @@ describe('ModuleCategory ', () => {
       // mouse enters category
       category.simulate('mouseEnter');
       // check: setOpenCategory() called from props
-      expect(setOpenCategory.mock.calls.length).toBe(3); // called twice before
+      expect(setOpenCategory.mock.calls.length).toBe(1); 
     });
   });
 
   describe('Library closed, category open', () => {
 
-    beforeEach(() => wrapper = mount(
-      <DragDropContextProvider backend={HTML5Backend}>
-        <ModuleCategory
-          name={"Add Data"}
-          modules={modules}
-          isReadOnly={false}
-          collapsed={false}
-          setOpenCategory={setOpenCategory}
-          libraryOpen={false}
+    beforeEach(() => {
+      setOpenCategory = jest.fn();
+      modules = [
+        <Module
+          key={"First Module"}
+          name={"First Module"}
+          icon={"add"}
+          id={88}
+          addModule={() => {}}
+          dropModule={() => {}}
+          isReadOnly={false} 
+          setOpenCategory={setOpenCategory}    
+        />,
+        <Module
+          key={"Second Module"}
+          name={"Second Module"}
+          icon={"url"}
+          id={101}
+          addModule={() => {}}
+          dropModule={() => {}}
+          isReadOnly={false} 
+          setOpenCategory={setOpenCategory}    
         />
-      </DragDropContextProvider>
-    ));
-    afterEach(() => wrapper.unmount());
+      ];
+      wrapper = mount(
+        <DragDropContextProvider backend={HTML5Backend}>
+          <ModuleCategory
+            name={"Add Data"}
+            modules={modules}
+            isReadOnly={false}
+            collapsed={false}
+            setOpenCategory={setOpenCategory}
+            libraryOpen={false}
+          />
+        </DragDropContextProvider>
+      )
+    });
+    afterEach(() => {
+      wrapper.unmount();
```

Comment:
may not be necessary, test?

Reply:
This is necessary because Jest can not have multiple HTML5 backends (due to wrapping the modules in a DragDropContextProvider)